domain/config.go

```diff
@@ -60,16 +60,126 @@ type Config struct {
 	CORS *CORSConfig `mapstructure:"cors"`
 }
 
+var (
+	DefaultConfig = Config{
```

Comment:
Note to reviewer: This is the new default configuration. Most values should be ported from the old `config.json`

Reply:
If we have both CLI parameters and environment variables, it can become a bit confusing.

In my opinion, environment variables (env vars) should have the highest priority. For example, Terraform prioritizes environment variables, giving them higher precedence, while CLI parameters have the lowest precedence (even lower than the config).

I think we should leave it as it is, **without** allowing the config to be changed via the CLI.

Reply:
That sounds good. Let's go with the current approach (no CLI parameters for now), and we can iterate in future PRs as needed.